File: src/main/java/com/example/Nadeuri/comment/CommentRepository.java
package com.example.Nadeuri.comment;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface CommentRepository extends JpaRepository<CommentEntity, Long> {

    // 특정 게시글에 달린 모든 댓글 조회
    List<CommentEntity> findByBoardId(Long boardId);

    // 특정 유저의 모든 댓글 조회 - userId
    List<CommentEntity> findByMemberUserId(String userId);

}

File: src/main/java/com/example/Nadeuri/comment/repository/CommentRepository.kt
package com.example.Nadeuri.comment.repository

import com.example.Nadeuri.comment.entity.CommentEntity
import org.springframework.data.jpa.repository.JpaRepository

interface CommentRepository : JpaRepository<CommentEntity, Long> {

    // 특정 게시글에 달린 모든 댓글 조회
    fun findByBoardId(boardId: Long): List<CommentEntity>

    // 특정 유저의 모든 댓글 조회 - userId
    fun findByMemberUserId(userId: String): List<CommentEntity>
}

/*
 댓글 관련 JPA Repository
 */